arches/install/arches-templates/project_name/src/declarations.d.ts

```diff
@@ -1,7 +1,8 @@
 // import declarations from other projects or Arches core
 import('../../node_modules/arches/arches/app/src/declarations.d.ts');
 
-// declare modules that have been added to your project (should mirror `package.json`)
+// declare untyped modules that have been added to your project in `package.json`
+// Module homepage on npmjs.com uses logos "TS" or "DT" to indicate if typed
```

Comment:
I didn't dive into the difference here, but if someone uses a moudle that's on DefinitelyTyped, the ts compiler provides a nice hint to install the types as a dev dependency a la https://github.com/archesproject/arches-dev-dependencies/pull/21, so I figured this is good enough.